### ISSUE TITLE
hsmcp.rb script has wrong permissions in RPM

### ISSUE THREAD

**Issue by paulmillar** [dCache/dcache#3158]
The file `/usr/share/dcache/lib/hsmcp.rb` is a script that dCache should run, but is not marked executable when installing from RPM

**Comment by femiadeyemi** [comment]
@jstarek is there a reason why this didn't go through the review channel?

**Comment by jstarek** [comment]
Yes, this PR was not meant to be merged (and the release team was aware of that). It only served as a means to debug an issue with Jenkins. As soon as that is understood, I'll post the patch to RB.

**Comment by jstarek** [comment]
https://rb.dcache.org/r/11956/

**Comment by kofemann** [comment]
fixed in 4.0
